### PR TITLE
Add documentation section to README

### PR DESCRIPTION
**Pull request by sloede**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,18 @@ To mesh a control file, type
...
+To get a list of the command line options available in HOHQMesh, type
+```bash
+./HOHQMesh -help
+```
+
+
+## Documentation
+The full documentation on how to use HOHQMesh, including the preparation of input files and
+the visualization of the resulting mesh files, can be found
+[here](https://trixi-framework.github.io/HOHQMesh).
```
Comment (ranocha) [inline README.md:107]:
```suggestion
[in the online documentation](https://trixi-framework.github.io/HOHQMesh).
```

Maybe?

Comment (andrewwinters5000) [inline README.md:107]:
```suggestion
Complete details on how to use HOHQMesh, including the preparation of input files,
the different formats of the resulting mesh files, and visualization instructions, can be found
[in the online documentation](https://trixi-framework.github.io/HOHQMesh).
```
Just to add a bit of clarity (including the suggestion from Hendrik).

File: README.md [hunk 2]
```diff
@@ -94,6 +94,25 @@ To mesh a control file, type
...
+```
+Examples/2D/GingerbreadMan/GingerbreadManMesh.mesh
+Examples/2D/GingerbreadMan/GingerbreadManPlot.tec
+Examples/2D/GingerbreadMan/GingerbreadManStats.txt
+```
+where the `.mesh` file stores the actual mesh, the `.tec` file is a Tecplot-compatible
+visualization file, and the `.txt` file contains statistical information on the mesh
+quality.
+
+The Tecplot file can be visualized, e.g., using the open source software
```
Comment (ranocha) [inline README.md:111]:
```suggestion
The Tecplot file can be visualized, e.g., using the open-source software
```

**Review by ranocha — APPROVED** [review]

**Review by DavidAKopriva — APPROVED** [review]
This looks good.

**Pull request merged**